pkg/interface/src/views/apps/chat/components/lib/chat-header.js

```diff
@@ -3,7 +3,7 @@ import { Link } from "react-router-dom";
 
 import { ChatTabBar } from "./chat-tabbar";
 import { SidebarSwitcher } from "../../../../components/SidebarSwitch";
-import { deSig } from "../../../../lib/util";
+import { deSig } from "../../../../../logic/lib/util";
```

Comment:
this is a bit deep in'it? maybe we should do something like this https://github.com/entwicklerstube/babel-plugin-root-import

Reply:
That would be a great quality of life change

Reply:
I'll PR after this gets merged